airbyte-cdk/bulk/toolkits/load-low-code/src/main/kotlin/io/airbyte/cdk/load/discoverer/operation/PropertyFactory.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.discoverer.operation
+
+import com.fasterxml.jackson.databind.JsonNode
+import io.airbyte.cdk.load.data.AirbyteType
+import java.util.function.Predicate
+
+class PropertyFactory(
```

Comment:
why is this one a factory as opposed to the supplier

Reply:
I don't know the naming that your team uses. Basically, this takes some API representation and generate the Kotlin domain objects. I think I've seen "adapter" elsewhere in the code so maybe that would be better? Anyway, I'm not attached to the naming here